Post/urls.py

```diff
@@ -0,0 +1,23 @@
+"""picATrip URL Configuration
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/3.1/topics/http/urls/
+Examples:
+Function views
+    1. Add an import:  from my_app import views
+    2. Add a URL to urlpatterns:  path('', views.home, name='home')
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+from django.urls import path
+from . import views
+
+urlpatterns = [
+    path('', views.PostListView.as_view(), name='view posts'),
+    # url on top of the relative path from the (localhost:8000/postlist)
+    path('<int:post_id>/', views.post_detail, name='post_detail'),
```

Comment:
Do you really need it? 
see in PicAtrip

Reply:
I wanted to make a separate comments page, in order to not change to postList page (since we divided our work in a certain way) and make it less clustered as if the comments will appear next to each post It will be less focused on the visibility of the image each post presents (will be updated in the future). We still need to implement the connection between the postlist page to the specific post page (which is made mainly for adding a comment at this pointt)